pymemcache/client/base.py

```diff
@@ -81,13 +81,26 @@ def _parse_hex(value):
 # Common helper functions.
 
 
-def _check_key(key, key_prefix=b''):
+def _bytearray(key):
```

Comment:
Using bytearray to bypass `if isinstance(key, VALID_STRING_TYPES):` doesn't make sense.